packages/editor-ui/src/constants.ts

```diff
@@ -444,4 +445,11 @@ export enum STORES {
 
 export const EXPRESSION_EDITOR_PARSER_TIMEOUT = 15_000; // ms
 
+export const KEEP_AUTH_IN_NDV_FOR_NODES = [
```

Comment:
What exactly? The whole list definition or the check if current node is in this list?
The list is extracted here since it used in different parts of the project (components, utils, ...)

Reply:
right but could this be a node property instead? in the node definition.. so that future nodes can easily be updated without having to update editor-ui

Reply:
Ah, well that's something we'll have to see with the nodes team. I can check that out with them.

Reply:
don't need to talk to anyone.. can just decide this ourselves